metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceOutput.pdl

```diff
@@ -41,7 +41,7 @@ record DataProcessInstanceOutput {
       "entityTypes": [ "dataset", "mlModel", "dataProcessInstance" ],
       "isUpstream": false,
       "isLineage": true,
-      "createdOn": "outputEdges/*/created/time"
+      "createdOn": "outputEdges/*/created/time",
       "createdActor": "outputEdges/*/created/actor"
```

Comment:
Needs commas on the rest of the fields too

Reply:
oh man how were we missing so many here!~